pcdsdevices/slits.py

```diff
@@ -243,10 +246,16 @@ def stage(self):
         """
         Store the initial values of the aperture position before scanning.
         """
-        self._original_vals[self.xwidth.setpoint] = self.xwidth.readback.get()
-        self._original_vals[self.ywidth.setpoint] = self.ywidth.readback.get()
+        self._pre_stage_gap = self.position
         return super().stage()
 
+    def unstage(self):
+        """
+        Restore the initial values of the aperture position.
+        """
+        self.move(self._pre_stage_gap[0], self._pre_stage_gap[1], wait=True)
```

Comment:
This seems like the right approach as compared to the old one 👍 

Not-important-but-a-notable-in-my-mind change: `_pre_stage_gap` doesn't seem to be set in `__init__` above, meaning this will raise a different exception if `unstage` is called before `stage` than before.

Reply:
Thanks, the old way was confusing, but perhaps more importantly, did not work as a `GroupDevice`

I think it's worth initializing the storage variable here. I'll revise appropriately.